CODEOWNERS

```diff
@@ -0,0 +1,5 @@
+## THIS SHOULD ALWAYS BE FIRST ##
+* @BitGo/btc-team
+
+.github @BitGo/btc-team @BitGo/infra
+renovate.json @BitGo/btc-team @BitGo/infra
```

Comment:
please also add en entry to define `@BitGo/git-admins` as owner of codeowners as well

Reply:
you mean like this ? Or it needs to have @BitGo/btc-team also. When i checked few other repositories i see mix use of @BitGo/git-admins or @BitGo/production. Trying to understand better so that i can plan next PR changes accordingly for other repositories when i work on CODEOWNERS alignment with renovate. 

```
# ALWAYS LAST
CODEOWNERS @BitGo/git-admins
```

Reply:
It should use git-admins. There are members of velocity (Eric) who is also a git-admin and he should be able to provide reviews on CODEOWNERS files as well.

Reply:
I actually have [this ticket](https://bitgoinc.atlassian.net/browse/VL-454) after a conversation with Hoffmann, which calls for replacing all uses of git-admins with production, and removing the git-admins team. I guess I forgot that last step :sweat_smile: 

Why is it a requirement that a non-btc team owns the codeowners file in this repository?

Reply:
This is a production control (not necessarily in all repositories, but most) as well as something that enforces our SDLC. In addition to the aforementioned reasons, it's a low-churn file. We discussed this as a team during DevOps West collaboration hours, and all agreed that `git-admins` or `production` should be the code owner of this file.